app/views/questionnaire.py

```diff
@@ -644,14 +644,28 @@ def _get_context(full_routing_path, block, current_location, form):
     if block['type'] == 'Summary':
         rendered_schema_json = renderer.render(g.schema.json, **schema_context)
         form = form or FlaskForm()
-        summary_rendered_context = build_summary_rendering_context(g.schema, rendered_schema_json, answer_store, metadata)
+        summary_rendered_context = build_summary_rendering_context(g.schema, rendered_schema_json, answer_store,
+                                                                   metadata)
         context = {'form': form,
                    'summary': summary_rendered_context,
                    'variables': variables,
                    'is_view_submission_response_enabled': is_view_submitted_response_enabled(g.schema.json),
                    'answers_are_editable': True}
         return context
 
+    elif block['type'] == 'SectionSummary':
```

Comment:
this is also very common. can you not reuse and then just have the logic around the line that builds the context?